code/modules/mob/living/silicon/ai/death.dm

```diff
@@ -31,6 +31,8 @@
 			SSshuttle.emergency.mode = SHUTTLE_DOCKED
 			SSshuttle.emergency.timer = world.time
 			priority_announce("Hostile enviroment resolved. You have 3 minutes to board the Emergency Shuttle.", null, 'sound/AI/shuttledock.ogg', "Priority")
+		for(var/obj/item/weapon/pinpointer/point in world)

```

Comment:
That is indeed a curated list.
You also need to have it remove itself on it's Destroy()